playwright/tests/meeting/agenda.spec.js

```diff
@@ -368,12 +368,18 @@ test.describe('past - desktop', () => {
     const materialsUrl = (new URL(event.agenda.url)).pathname
     const materialsInfo = {
       url: event.agenda.url,
-      slides: _.times(5, idx => ({
-        id: 100000 + idx,
-        title: faker.commerce.productName(),
-        url: `/meeting/${meetingData.meeting.number}/materials/slides-${meetingData.meeting.number}-${event.acronym}-${faker.internet.domainWord()}`,
-        ext: ['pdf', 'html', 'md', 'txt', 'pptx'][idx]
-      })),
+      slides: {
+        decks: _.times(5, idx => ({
+          id: 100000 + idx,
+          title: faker.commerce.productName(),
+          url: `/meeting/${meetingData.meeting.number}/materials/slides-${meetingData.meeting.number}-${event.acronym}-${faker.internet.domainWord()}`,
+          ext: ['pdf', 'html', 'md', 'txt', 'pptx'][idx]
+        })),
+        actions: [{
+          label: 'Propose slides',
+          url: '/meeting/${meetingData.meeting.number}/session/${event.sessionId}/propose_slides'
```

Comment:
Should use backticks `, not single quotes, otherwise the meeting number and session id won't be replaced in the string.

Reply:
Fixed